tests/MenderAPI/__init__.py

```diff
@@ -10,6 +10,11 @@
 # don't complain about non-verified ssl connections
 requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
 
+# Define get funtion before the below imports to avoid circular dependency
```

Comment:
The "correct" way to avoid circular dependency is to move the global and the getter/setter method to a dedicated file which can be imported multiple times by the consumers.

Reply:
Thanks.

In here I tried to do the minimum changes as the full `MenderAPI` module is the next thing to refactor. The current design is very troublesome (mainly because of the global scope).